cmd/kube-proxy/app/server.go

```diff
@@ -203,6 +203,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.config.EnableProfiling, "profiling", o.config.EnableProfiling, "If true enables profiling via web interface on /debug/pprof handler.")
 
 	fs.Float32Var(&o.config.ClientConnection.QPS, "kube-api-qps", o.config.ClientConnection.QPS, "QPS to use while talking with kubernetes apiserver")
+	fs.Var(&o.config.DetectLocal, "detect-local", "Mode to use to detect local traffic")
```

Comment:
I naturally assume `detect-local` would be a bool flag, wondering if `--detect-local-mode` is a better name?

Reply:
good point. I'll change it to --detect-local-mode.

Reply:
updated to --detect-local-mode